src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -35,11 +37,30 @@ const generateInitialOptions = series => {
             ).toFixed(2);
 
             tooltipData.push(`Δ ${delta} (${deltaPercentage}%)`);
+
```

Comment:
this whole `tooltips.callbacks.footer` method should go into the `CustomTooltip`